tmessage/cli.py

```diff
@@ -1,6 +1,7 @@
 import paho.mqtt.client as mqtt
 import argparse
 from colorama import init, deinit, Fore, Back, Style
+import auth  # auth.py

```

Comment:
@vitokhangnguyen Here I am getting this error `ModuleNotFoundError: No module named 'auth'` when I start the program using tmessage command i.e `tmessage --user test` but it is working perfectly fine when I run using the command `python tmessage/cli.py --user test`.

Reply:
Hi,
I fixed the issue by changing `import auth` to...
```python
import tmessage.auth as auth
```
It is working now but I don't fully understand what is the different and why it complains in 1 case but not the other, it would be nice if you can enlighten me :)